components/director/internal/open_resource_discovery/validation_test.go

```diff
@@ -3304,6 +3581,31 @@ func TestDocuments_ValidateVendor(t *testing.T) {
 				doc := fixORDDocument()
 				doc.Vendors[0].Labels = json.RawMessage(invalidLabelsWhenKeyIsWrong)
 
+				return []*open_resource_discovery.Document{doc}
+			},
+		},
+		{
```

Comment:
New test case must be added where a value of the `Partners` array doesn't match the required regex